test/with_dummyserver/test_connectionpool.py

```diff
@@ -805,6 +805,14 @@ def test_multi_redirect_history(self):
             (302, '/multi_redirect?redirect_codes=200')
         ])
 
+    def test_retry_after(self):
+        r = self.pool.request('GET', '/retry_after')
+        r = self.pool.request('GET', '/retry_after')
+        self.assertEqual(r.status, 429)
```

Comment:
To get a 429 response. The server responses 429 if a client requests twice in a second.